Flixter/GridViewController.m

```diff
@@ -0,0 +1,108 @@
+//
+//  CollectionsViewController.m
+//  Flixter
+//
+//  Created by Angelina Zhang on 6/17/22.
+//
+
+#import "GridViewController.h"
+#import "MoviePosterCollectionCell.h"
+#import "DetailsViewController.h"
+#import "UIImageView+AFNetworking.h"
+
+@interface GridViewController () <UICollectionViewDataSource, UICollectionViewDelegate, UICollectionViewDelegateFlowLayout>
+
+@property (nonatomic, strong) NSArray *movieData;
+@property (weak, nonatomic) IBOutlet UICollectionView *moviesCollectionView;
+@property (weak, nonatomic) IBOutlet UIActivityIndicatorView *loadingIndicator;
+
+@end
+
+@implementation GridViewController
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    [self.loadingIndicator startAnimating];
+    self.moviesCollectionView.dataSource = self;
+    self.moviesCollectionView.delegate = self;
+    [self.loadingIndicator startAnimating];
+    
+    UIRefreshControl *refreshControl = [[UIRefreshControl alloc] init];
+    [refreshControl addTarget:self action:@selector(beginRefresh:) forControlEvents:UIControlEventValueChanged];
+    [self.moviesCollectionView insertSubview:refreshControl atIndex:0];
+
+    [self networkRequest];
+    // Do any additional setup after loading the view.
```

Comment:
This is why I always delete boilerplate before I start writing. Too easy to forget and leave it in.